### PR TITLE
Update language_pl.h

### PR DESCRIPTION
**Pull request by android444**
Translation of messages in English to Polish.

### CODE REVIEW

File: Marlin/language_pl.h [hunk 1]
```diff
@@ -159,13 +159,13 @@
 #define MSG_THERMAL_RUNAWAY                 "Zanik temp."
 #define MSG_ERR_MAXTEMP                     "Err: max. temp."
 #define MSG_ERR_MINTEMP                     "Err: min. temp."
-#define MSG_ERR_MAXTEMP_BED                 "Err: max. temp. loza"
-#define MSG_ERR_MINTEMP_BED                 "Err: min. temp. loza"
+#define MSG_ERR_MAXTEMP_BED                 "Err: max. temp. stolu"
+#define MSG_ERR_MINTEMP_BED                 "Err: min. temp. stolu"
```
Comment (thinkyhead) [inline Marlin/language_pl.h]: One character too long. Must be 20 characters or fewer. 18 or fewer is even better.

File: Marlin/language_pl.h [hunk 2]
```diff
@@ -177,68 +177,68 @@
...
 #if LCD_WIDTH >= 20
-  #define MSG_INFO_PRINT_COUNT              "Print Count"
-  #define MSG_INFO_COMPLETED_PRINTS         "Completed"
-  #define MSG_INFO_PRINT_TIME               "Total print time"
-  #define MSG_INFO_PRINT_LONGEST            "Longest job time"
-  #define MSG_INFO_PRINT_FILAMENT           "Extruded total"
+  #define MSG_INFO_PRINT_COUNT              "Licznik wydrukow"
+  #define MSG_INFO_COMPLETED_PRINTS         "Ukończono"
+  #define MSG_INFO_PRINT_TIME               "Calkow. czad druku"
+  #define MSG_INFO_PRINT_LONGEST            "Najdluzszy czas druku"
```
Comment (thinkyhead) [inline Marlin/language_pl.h]: Too long.

Comment (android444) [reply Marlin/language_pl.h]: Change all subtitles?
Maximum is 18 characters?

**Comment by thinkyhead** [comment]
Please rebase onto the latest `RCBugFix` and squash commits.

**Comment by android444** [comment]
ok

**Comment by thinkyhead** [comment]
You don't have to close a PR to update it. But if it's easier for you, fine, create a new one.

**Comment by android444** [comment]
Sorry.

**Comment by thinkyhead** [comment]
> Sorry.

No problem! PRs are something to get used to.

Are you able to access command-line git and squash commits? As in…
```
git rebase -i HEAD~5
```

**Comment by thinkyhead** [comment]
Superseded by #5410

**Pull request closed without merging**